clients/imodelhub/src/imodelhub/Events.ts

```diff
@@ -79,7 +80,7 @@ export abstract class IModelHubEvent extends IModelHubBaseEvent {
 }
 
 /** Base type for iModelHub events that have BriefcaseId.
- * @beta
+ * @public

```

Comment:
Going with the conservative approach here (which I like), should we make this internal too?